src/views/components/shared/InputField/component.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { getIn } from 'formik'
+
+import Input from '../Input'
+
+import {
+  propTypes as inputPropTypes,
+  defaultProps as inputDefaultProps
+} from '../Input/types'
+
+const InputField = ({
+  field: { name, onChange, onBlur, value },
+  form: { touched, errors },
+  placeholder,
+  type,
+  prefix
+}) => {
+  const fieldTouched = getIn(touched, name)
+  const fieldErrors = getIn(errors, name)
```

Comment:
move this to the getters inside container

Reply:
Refactored